openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OAlgoUtils.java

```diff
@@ -108,8 +103,6 @@ protected final T parseParams(final Frame trainingFrame,
      *
      * @param h2oParams concrete implementation of {@link ModelParametersSchemaV3 algorithm params}.
      * @return concrete implementation of {@link ModelBuilder H20 Model Builder} created from the h2oParams.
-     *
-     * @since 1.2.1
```

Comment:
revert this deletion and below one that state since 1.2.1.
the reason is that these methods already existed in 1.2.1, they weren't introduced in 0.1.0